frontend/test/metabase/scenarios/admin/settings/sso/google.cy.spec.js

```diff
@@ -8,16 +8,30 @@ describe("scenarios > admin > settings > SSO > Google", () => {
   });
 
   it("Google sign-in client ID should save on subsequent tries (metabase#15974)", () => {
-    cy.findByLabelText("Client ID").type("123");
+    cy.findByLabelText("Client ID").type(
+      "fake-client-id.apps.googleusercontent.com",
+    );
     saveSettings();
 
     // This string lingers for far too long in the UI, so we have to wait for it to disappear before we assert on that same button again.
     // Otherwise, the test fails. That's why we added a custom timeout of 6s.
     cy.findByText("Success", { timeout: 6000 }).should("not.exist");
 
-    cy.findByDisplayValue("123").type("456");
+    cy.findByDisplayValue("fake-client-id.apps.googleusercontent.com").type(
+      "fake-client-id2.apps.googleusercontent.com",
+    );
     saveSettings();
   });
+
+  it(
+    "Google sign-in client ID form should show an error message if it does not end with the correct suffix (metabase#15975)",
+  ),
+    () => {
+      cy.findByLabelText("Client ID").type("fake-client-id");
+      saveSettings();
+
+      cy.findByText("Invalid Google Sign-In Client ID");
+    };
```

Comment:
@noahmoss I didn't notice this before, but this test never runs. The arrow function should be the second argument of the `it()` function like this:
```js
it("should do somehting", ()=> {});
```

Reply:
Oh you're right. Seems like it's already been fixed?